pkg/api/types.go

```diff
@@ -3312,6 +3314,31 @@ const (
 	TLSCertKey = "tls.crt"
 	// TLSPrivateKeyKey is the key for the private key field in a TLS secret.
 	TLSPrivateKeyKey = "tls.key"
+
+	// SecretBootstrapToken is used during the automated bootstrap process (first
+	// implemented by kubeadm). It stores tokens that are used to sign well known
+	// ConfigMaps. They may also eventually be used for authentication.
+	SecretTypeBootstrapToken SecretType = "bootstrap.kubernetes.io/token"
```

Comment:
@smarterclayton So you're saying this should be in `cmd/kubeadm` only?
What about if kops wants to use this as well?

Best option might be to make a library that's consuming this functionality in controller-manager (e.g. `pkg/bootstrap/token` or similar), have that constant there and then consume that lib from kubeadm (and maybe other binaries). WDYT about that?